index.html

```diff
@@ -421,12 +421,48 @@
 				<h3>Default Reading Order</h3>
 
 				<p>The default reading order MUST include at least one <a>primary resource</a>.</p>
+				
 			</section>
 
 			<section id="table-of-contents">
 				<h3>Table of Contents</h3>
 
-				<div class="ednote">Placeholder for identifying table of contents - whether embedded or by reference.</div>
+
+				<p>The table of contents is recommended in the infoset.</p>
+				
+				<p>
+					An author MAY omit the table of contents from the <a href="#manifest">manifest</a> if it is included in an HTML <code>nav</code> element identified by the role <code>doc-toc</code>&nbsp;[[!dpub-aria-1.0]]. 
```

Comment:
The processing rules should be the following:

1. Look for a TOC in the manifest
2. If we cannot find a TOC in the manifest, attempt to identify a primary or secondary resource (based on a rel, or something equivalent in our syntax)
3. If we cannot identify a primary or secondary resource we could then trigger a failure mode:
    1. scan all resources for a `nav@role="doc-toc"` or a `link@rel="contents"`
    2. or default to a well-known location (I'm not a fan)
    3. or extract an outline using HTML elements